bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -91,6 +93,7 @@ export default injectIntl(withModalMounter(withTracker(({ intl, baseControls })
     chatIsOpen: Session.equals('openPanel', 'chat'),
     openPanel: Session.get('openPanel'),
     userListIsOpen: !Session.equals('openPanel', ''),
+    UserInfo,
```

Comment:
We are in `withTracker` block. Won't returning UserInfo cause rerendering on any [subtle] change within UserInfo?